go-controller/pkg/libovsdb/ops/acl_test.go

```diff
@@ -116,9 +116,16 @@ func TestCreateOrUpdateACL(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.desc, func(t *testing.T) {
+			sw := &nbdb.LogicalSwitch{
+				Name: "sw1",
+				UUID: buildNamedUUID(),
+				ACLs: []string{tt.initialACL.UUID},
+			}
+
 			nbClient, cleanup, err := libovsdbtest.NewNBTestHarness(libovsdbtest.TestSetup{
 				NBData: []libovsdbtest.TestData{
-					tt.initialACL,
+					sw.DeepCopy(),
+					tt.initialACL.DeepCopy(),
```

Comment:
why do you need to deepcopy all the db objects here?

Reply:
Not needed. There was a time when initializing the test harness did not copy so this was needed. Not anymore, probably an instinct reaction of mine when troubleshooting something else. Will undo.